### PR TITLE
Update README, Add Makefile and minor tweaks

### PR DESCRIPTION
**Pull request by rn**
- Update documentation. Describe how to use and build the images.
- Add a Makefile to build the artefacts
- Add .gitignore
- Minor tweak to `init-lcow` and updated package

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+build: lcow.yml Makefile
```
Comment (justincormack) [inline Makefile:2]: should have `.PHONY: build`

Comment (justincormack) [reply Makefile:2]: (and for release)

**Comment by justincormack** [comment]
minor thing but otherwise LGTM

File: Makefile (hunk 1 above)
Comment (rn) [reply Makefile:2]: fixed. Thanks

**Pull request merged**